joomla/joomla-cms
# [4.0][a11y] Focus highlighting

Pull Request for Issue #20570 .

### Summary of Changes

- Add required rule for CSS focus-visible polyfil
- Remove Flying Focus CSS from Atum
- Remove Flying Focus CSS from Cassiopeia
- Add default focus styling to Atum (overrides browser defaults for consistency across all browsers).


### Testing Instructions

- Tab through links ensuring focus highlighting appears the same across all browsers (not buttons)
- Click down on link, drag mouse away and let go. Ensure the item is not highlighted. This will test the focus-visible polyfil

![focusf](https://user-images.githubusercontent.com/2803503/40842608-5348c058-65a6-11e8-844b-fcfe8ff60d0a.gif)
